src/cli/onefuzz/api.py

```diff
@@ -970,6 +970,58 @@ def list(
             results[container] = self.onefuzz.containers.files.list(container).files
         return results
 
+    def delete(self, job_id: UUID_EXPANSION, *, only_job_specific: bool = True) -> None:
```

Comment:
Could we make this support a dry-run mode, without too much pain?

Reply:
added